src/main/java/uk/gov/justice/probation/courtcaseservice/service/HearingEntityInitService.java

```diff
@@ -72,8 +84,9 @@ public Optional<HearingEntity> findFirstByHearingIdFileUpload(String hearingId)
     }
 
     @Transactional
-    public Optional<HearingEntity> findHearingByHearingIdAndDefendantIdInitialiseCaseDefendants(String hearingId, String defendantId) {
-        var hearing = hearingRepository.findFirstByHearingId(hearingId);
+    public Optional<HearingEntity> findAllHearingByHearingIdAndDefendantIdInitialiseCaseDefendants(String hearingId, String defendantId) {
```

Comment:
hi, it says findAll but the function returns 1 HearingEntity. Should it be `findHearingByHearingIdAndDefendantIdInitialiseCaseDefendants` ?